dashboards/google-internal-tcp-lb-rule/internal-tcp-lb-rule.json

```diff
@@ -304,12 +300,26 @@
             ],
             "thresholds": [],
             "yAxis": {
-              "label": "",
               "scale": "LINEAR"
             }
           }
         }
       }
     ]
-  }
+  },
+  "legacyCategory": "integration",
```

Comment:
@philshin-google @SamiSousa looks like this unexpected field is breaking dashboard validation in the copybara pipeline: sponge2/c9585213-ea7a-4b9e-a7d2-f6257a0c2897

Also visible when you copy and paste this dashboard JSON into the dashboard builder: 
![Screenshot 2025-01-03 at 4 15 08 PM](https://github.com/user-attachments/assets/6d343a6e-85b6-4e12-a81e-0ae526b0354e)

Reply:
Fix at https://github.com/GoogleCloudPlatform/monitoring-dashboard-samples/pull/900